cert-manager-1.17.yaml

```diff
@@ -69,6 +69,22 @@ subpackages:
       provides:
         - cert-manager-controller=${{package.full-version}}
 
+  - name: cert-manager-controller-${{vars.major-minor-version}}-bitnami-compat
+    dependencies:
+      provides:
+        - cert-manager-controller-bitnami-compat=${{package.full-version}}
+    pipeline:
+      - uses: bitnami/compat
```

Comment:
We should consider dropping the bitnami/compat part as it doesn't look to need any of the files being pulled in

Reply:
I'm keeping it here to keep the licenses file intact that's coming along but no harm in removing as well.

Reply:
If it's just a symlink we're creating, i'd drop this, as we're copying in stuff we don't need / aren't used, but not blocking